exchanges/okx/okx_types.go

```diff
@@ -3039,14 +3039,14 @@ type SubAccountAPIKeyResponse struct {
 type MarginBalanceParam struct {
 	AlgoID     string  `json:"algoId"`
 	Type       string  `json:"type"`
-	Amount     float64 `json:"amt,string"`               // Adjust margin balance amount Either amt or percent is required.
-	Percentage float64 `json:"percent,string,omitempty"` // Adjust margin balance percentage, used In Adjusting margin balance
+	Amount     float64 `json:"amt,string"`     // Adjust margin balance amount Either amt or percent is required.
+	Percentage float64 `json:"percent,string"` // Adjust margin balance percentage, used In Adjusting margin balance
```

Comment:
just checking Percentage is a requirement?

Reply:
that was a typo, sorry. I have fixed it